src/function/aggregate/nested/histogram.cpp

```diff
@@ -48,8 +48,12 @@ static void HistogramUpdateFunction(Vector inputs[], FunctionData *, idx_t input
 			if (!state->hist) {
 				state->hist = new map<T, idx_t>();
 			}
+			// TODO: change this again once the cause of the heap buffer overflow is found
 			auto value = (T *)input_data.data;
-			(*state->hist)[value[input_data.sel->get_index(i)]]++;
+			auto idx = input_data.sel->get_index(i);
+			auto key = value[idx];
+			(*state->hist)[key];
```

Comment:
I don't believe this line is [necessary](https://www.cplusplus.com/reference/map/map/operator[]/)?

Reply:
Ah yes it is not, I accidentally did not remove this after debugging...